lib/cosmos/packets/packet.rb

```diff
@@ -165,6 +165,53 @@ def limits_change_callback=(limits_change_callback)
       end
     end
 
+    # Review bit offset to look for overlapping definitions. This will allow
+    # gaps in the packet, but not allow the same bits to be used for multiple
+    # variables.
+    #
+    # @return [Array<String>] Warning messages for big definition overlaps
+    def check_bit_offsets
```

Comment:
This was previously in packet_config but I felt like it belongs here inside the Packet class.